src/wfcomman.c

```diff
@@ -1075,6 +1075,28 @@ AppCommandProc(register DWORD id)
       ret = ExecProgram(szToRun, szParams, szDir, FALSE, bRunAs);
       LocalFree(szDir);
    }
+   break;
+
+   case IDM_STARTEXPLORER:
+   {
+       BOOL bDir;
+       LPTSTR szDir;
+       TCHAR szToRun[MAXPATHLEN];
+
+       szDir = GetSelection(1 | 4 | 16, &bDir);
+       if (!bDir && szDir)
+           StripFilespec(szDir);
+
+       if (GetSystemDirectory(szToRun, MAXPATHLEN) != 0)
+           lstrcat(szToRun, TEXT("\\..\\explorer.exe"));
+       else
+           lstrcpy(szToRun, TEXT("explorer.exe"));
+
+       TCHAR szParams[MAXPATHLEN] = { TEXT('\0') };
+
+       ret = ExecProgram(szToRun, szDir, szDir, FALSE, GetKeyState(VK_SHIFT) < 0);

```

Comment:
It doesn't matter, but what's the scenario for shift support? Normally there's only one explorer process, so elevating here will prompt the user but end up launching a window in the context of the unelevated explorer process (it will be functionally identical.)

Reply:
Good point, Will remove it